parquet-hadoop/src/test/java/org/apache/parquet/hadoop/example/TestInputOutputFormat.java

```diff
@@ -66,7 +66,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class TestInputOutputFormat {
+public class
+TestInputOutputFormat {
```

Comment:
Modified by accident?

Reply:
Yes, thanks for the noticing.

Reply:
Looking good on my side. Thanks @gszadovszky